lib/rules.js

```diff
@@ -353,6 +353,267 @@ export const GROUP_TERMS_BY_ROOT = defineRule(
 // e.g nthRoot(9, 2) -> 3
 export const NTH_ROOT_VALUE = defineRuleString('nthRoot(#a, #b)', '#eval(nthRoot(#a, #b))', {a: query.isNumber, b: query.isNumber})
 
+// FACTOR
```

Comment:
Can you put these in their own file?  I'd like eventually rename this file to be `simple-rules.js` and have it contain only rules which can be full expressed with patterns.